package.json

```diff
@@ -2,14 +2,18 @@
   "name": "data-layer-helper",
   "version": "0.1.0",
   "devDependencies": {
+    "eslint": "^7.3.1",
+    "eslint-config-closure-es6": "^0.1.1",
+    "eslint-config-google": "^0.14.0",
+    "eslint-plugin-jasmine": "^4.1.1",
     "grunt": "~0.4.1",
+    "grunt-closure-linter": "~0.1.0",
```

Comment:
In this case, should the old/deprecated closure linter get removed?

Reply:
I guess this is a pretty good place to do that, I removed it and it's usages.